src/Sylius/Behat/Context/Setup/ProductAttributeContext.php

```diff
@@ -140,37 +147,67 @@ public function thisAttributeHasSetMinValueAsAndMaxValueAs(ProductAttributeInter
     }
 
     /**
-     * @Given /^(this product) has (.+?) attribute "([^"]+)" with values "([^"]+)", "([^"]+)"$/
+     * @Given /^(this product) has select attribute "([^"]+)" with value "([^"]+)"$/
      */
-    public function thisProductHasSelectAttributeWithValues(
+    public function thisProductHasSelectAttributeWithValue(
         ProductInterface $product,
-        $productAttributeType,
-        $productAttributeName,
-        $firstAttributeValue,
-        $secondAttributeValue,
-        $language = 'en_US'
+        string $productAttributeName,
+        string $attributeValue
     ) {
+        $values = [$this->faker->uuid => $attributeValue];
+
+        $this->createSelectProductAttributeValue($product, $productAttributeName, $values);
+    }
+
+    /**
+     * @Given /^(this product) has select attribute "([^"]+)" with values "([^"]+)" and "([^"]+)"$/
```

Comment:
I guess it can be merged through variadics support with the previous step: 
`@Given /^(this product) has select attribute "([^"]+)" with value "([^"]+)"$/`